packages/jest-snapshot/src/State.js

```diff
@@ -67,22 +74,42 @@ export default class SnapshotState {
     });
   }
 
-  _addSnapshot(key: string, receivedSerialized: string) {
+  _addSnapshot(key: string, receivedSerialized: string, isInline: boolean) {
     this._dirty = true;
-    this._snapshotData[key] = receivedSerialized;
+    if (isInline) {
+      const stack = new Error().stack.split(/\n/);
```

Comment:
We can use `callsites` here, but it's way easier with sourcemaps to use errors.

We have an error already from the matcher on `expect`, but we might not pass it through?